10up/ElasticPress
# Rename wp-cli commands

### Description of the Change
This PR is used to rename the WP-CLI commands as requested in #2981 


Closes #2981 

### How to test the Change
- Run the old WP-CLI commands, they will give a deprecation warning along with the expected output
- Run the renamed commands, they will give the expected output


### Changelog Entry

> Added - `get-cluster-indices`, `get-indices`, `sync`, `clear-sync`, `get-ongoing-sync-status`, `get-last-cli-sync`, `stop-sync` WP-CLI commands
> Deprecated - `get-cluster-indexes`, `get-indexes`, `index`, `clear-index`, `get-indexing-status`, `get-last-cli-index`, `stop-indexing` WP-CLI commands

### Credits

Props @MARQAS 

### Checklist:
